src/main/java/de/christophkraemer/rhino/javascript/RhinoScriptEngineFactory.java

```diff
@@ -133,7 +138,7 @@ public String getProgram(String... statements) {
         names = new ArrayList<String>(6);
         names.add("js");
         names.add("rhino");
-        names.add("rhino17R5");
```

Comment:
Hi,

The point was to have a unique name, which will prevent us from the need of releasing a new version of the rhino-script-engine for each release of a new Rhino version.
If you like, we can let "rhino17R5" and add a more specific name, we could use "de.christophkraemer.mozilla.rhino" or "de.christophkraemer.rhino".